plugin/CactbotEventSource/CactbotEventSource.cs

```diff
@@ -47,6 +47,7 @@ public class CactbotEventSource : EventSourceBase, ILogger {
     private FateWatcher fate_watcher_;
 
     private string language_ = null;
+    private string pc_locale = null;
```

Comment:
Can you name this `pc_locale_` with an underscore, to match the other member variables?